components/event-publisher-proxy/cmd/event-publisher-proxy/nats/nats.go

```diff
@@ -85,12 +83,7 @@ func (c *Commander) Start() error {
 	defer connection.Close()
 
 	// configure the message sender
-	var messageSenderToNATS sender.GenericSender
-	if c.jetstreamMode {
-		messageSenderToNATS = sender.NewJetStreamMessageSender(ctx, connection, c.envCfg, c.logger)
-	} else {
-		messageSenderToNATS = sender.NewNATSMessageSender(ctx, connection, c.logger)
-	}
+	messageSenderToNATS := sender.NewJetStreamMessageSender(ctx, connection, c.envCfg, c.logger)
```

Comment:
one general thing, sometimes we the word `JetStream` in the variable and file names, sometimes the word `nats`. Can we go with one consistent word? both are fine for me, but `nats` would be better, since JetStream is a NATS.